app/src/main/java/ar/com/wolox/android/example/BootstrapApplication.kt

```diff
@@ -18,8 +19,14 @@ import okhttp3.logging.HttpLoggingInterceptor.Level
 
 class BootstrapApplication : WolmoApplication() {
 
+    lateinit var sharedPreferences: SharedPreferences
+    lateinit var sharePref: SharedPreferencesManager
+    lateinit var userSession: UserSession
     override fun onInit() {
         // Initialize Application stuff here
+        sharedPreferences = getSharedPreferences(getString(R.string.prefs_name), Context.MODE_PRIVATE)
+        sharePref = SharedPreferencesManager(sharedPreferences)
+        userSession = UserSession(sharePref)
```

Comment:
We talked about this in private. You don't need to instantiate UserSession. You can inject it by writing `@Inject` when declaring this property.